fastlane/Fastfile

```diff
@@ -99,6 +102,9 @@ platform :ios do
       podfile: "Internal/Debug App/Appetize Podfile/Podfile"
     )
 
+    # Run Unit Tests
+    tests
```

Comment:
Does the `tests` step not build anyway? 

If so we could remove the `build_app` step below

Reply:
Good shout, let me check. If the `run_tests` lane can take all of the params `build_app` does, we can maybe reduce this to one step.